dspot/src/main/java/fr/inria/diversify/utils/AmplificationHelper.java

```diff
@@ -367,9 +370,27 @@ public static void prepareTestMethod(CtMethod cloned_method, Factory factory) {
         testAnnotation.setAnnotationType(ref);
 
         Map<String, Object> elementValue = new HashMap<>();
-        elementValue.put("timeout", timeOutInMs);
+        java.lang.annotation.Annotation originalTestAnnotation = cloned_method.getAnnotation(org.junit.Test.class);
```

Comment:
This might not work if the junit version of the program under amplification is different than the version that **DSpot** uses.
You can reuse:

```java
 CtAnnotation testAnnotation = cloned_method.getAnnotations().stream()
                .filter(annotation -> annotation.toString().contains("Test"))
                .findFirst().orElse(null);
```
And works on **Spoon** nodes instead of the `java.lang.annotation.Annotation`.
Also, you don't have to re-add the `expected` values. You can just add the `timeout` value, using `testAnnotation.addValue("timeout", factory.createLiteral(10000))`
, right?